utils/db.ts

```diff
@@ -0,0 +1,146 @@
+// Copyright 2023 the Deno authors. All rights reserved. MIT license.
+export const kv = await Deno.openKv();
+
+const versionstamp = null;
+
+export interface InitItemValue {
+  userId: string;
+  title: string;
+  url: string;
+}
+
+export async function createItem(initItem: InitItemValue) {
+  const itemId = crypto.randomUUID();
+
+  const itemKey = ["items", crypto.randomUUID()];
+  const itemsByUserKey = ["items_by_user", initItem.userId, itemId];
+
+  const item: ItemValue = { ...initItem, score: 0, createdAt: new Date() };
+  const res = await kv.atomic()
+    .check({ key: itemKey, versionstamp })
+    .check({ key: itemsByUserKey, versionstamp })
```

Comment:
Where is `versionstamp` coming from here? Did you mean to use `versionstamp: null`?

Reply:
Yes, I'm using `versionstamp = null` for the time being for all instances of `versionstamp` within this file. Is it fine if I instead use some arbitrary value like `00000000000000010000` or similar?

Reply:
Done. Though, why inline over global? I assume it's because there's nothing in common between the various `versionstamp`s throughout the file. Having a single global suggests that there is and can be misleading. Is that right?